spi-fly/spi-fly-dynamic-bundle/pom.xml

```diff
@@ -51,8 +51,8 @@
     <dependencies>
         <dependency>
             <groupId>org.ow2.asm</groupId>
-            <artifactId>asm-debug-all</artifactId>
-            <version>5.0.3</version>
+            <artifactId>asm-all</artifactId>
+            <version>6.0_ALPHA</version>
```

Comment:
Agreed. I suggest a compromise for the meantime. Let's wait for a release of SPI Fly until ASM 6.0 gets release. I have updated my PR and migrated to the latest stable ASM 5.x package (version 5.2). A modification of the import package upper range makes SPI Fly upward compatibility to the upcoming ASM 6.x package.

What do you think?